src/predefined-variables.ts

```diff
@@ -25,6 +26,9 @@ export function init ({gitData, argv}: PredefinedVariablesOpts): {[name: string]
         CI_COMMIT_BRANCH: gitData.commit.REF_NAME, // Not available in merge request or tag pipelines
         CI_COMMIT_REF_NAME: gitData.commit.REF_NAME, // Tag or branch name
         CI_COMMIT_REF_SLUG: gitData.commit.REF_NAME.replace(/[^a-z\d]+/ig, "-").replace(/^-/, "").slice(0, 63).replace(/-$/, "").toLowerCase(),
+        CI_COMMIT_TIMESTAMP: gitData.commit.TIMESTAMP,
```

Comment:
https://github.com/firecow/gitlab-ci-local/blob/56319b8d8df4f1227653817a900f97bf31bd309c/src/git-data.ts#L33

Do you need a proper timestamp ?  `CI_COMMIT_TIMESTAMP` currently, it's kinda "mock"

Reply:
Depends on how we we look at it, i think the current timestamp actually can make sense too. lets leave it haha

Reply:
I am not sure as well :sweat_smile: but given that it's possible to override it, maybe it's better to have at least something by default